sample/src/main/java/com/google/accompanist/sample/adaptive/NavDrawerFoldAwareColumnSample.kt

```diff
@@ -68,7 +67,6 @@ class NavDrawerFoldAwareColumnSample : ComponentActivity() {
 @Composable
 fun NavDrawerExample(activity: Activity) {
     val icons = listOf(
-        Icons.Default.List,
```

Comment:
Nit: Was this removed? Or was this one of the `AutoMirrored` icons?

Reply:
It was automirrored but the label looks weird in the sample because of splitting at the first `.`. It seemed easier to just remove it rather than fix that